src/containers/SearchPage/README.md

```diff
@@ -36,20 +36,24 @@ possible for Topbar to take current filters into account.
 
 ## MainPanel
 
-MainPanel has two functions: showing searchResults and showing filters. There are two sets of
-filters that can be passed to it: `primaryFilters`, `secondaryFilters`.
+MainPanel has two functions: showing searchResults and showing filters. Filters are primarily added,
+removed, reordered and configured through _marketplace-custom-config.js_
 
-Primary filters are filters that are shown always on top of SearchResults as dropdown-selections. We
-recommend that only 1 - 3 primary filters are passed in since they start to take too much space on
-narrow screens.
+```shell
+└── src
+    └── marketplace-custom-config.js
+```
+
+There you can set some filters to be primary filters. They are shown always on top of SearchResults
+as dropdown-selections on Desktop layout. We recommend that only 1 - 3 primary filters are passed in
+since they start to take too much space on narrow screens.
```

Comment:
Now I see that the docs are here. Maybe refer to this file in the custom config?